modules/library/main/src/main/java/org/geotools/filter/function/JsonPointerFunction.java

```diff
@@ -152,4 +154,9 @@ private void serializeObject(JsonParser parser, JsonGenerator generator) throws
         }
         generator.writeEndObject();
     }
+
+    @Override
+    public Object accept(ExpressionVisitor visitor, Object extraData) {
```

Comment:
Does the same as the overridden method, should be removed.

Reply:
fixed